src/commands/build-image.yml

```diff
@@ -37,32 +37,18 @@ parameters:
     default: ""
     description: >
       Extra flags to pass to docker build. For examples, see
-      https://docs.docker.com/engine/reference/commandline/build
-
-  checkout:
-    type: boolean
-    default: false
-    description: >
-      Boolean for whether or not to checkout as a first step. Default is false.
-
-  ecr-login:
-    type: boolean
-    default: false
-    description: >
-      Boolean for whether or not to log in to ECR before building. Default is false.
+      https://docs.docker.com/engine/reference/commandline/buildx
 
   skip-when-tags-exist:
     type: boolean
     default: false
     description: Whether to skip image building if all specified tags already exist in ECR
 
   region:
-    type: env_var_name
-    default: AWS_REGION
+    type: string
+    default: "us-west-2"
```

Comment:
We should not be setting any default regions for users. This should likely be `${AWS_DEFAILT_REGION}` and documented, or no default value should be given.